lib/simplifyExpression/basicsSearch/index.js

```diff
@@ -19,6 +20,8 @@ const removeMultiplicationByOne = require('./removeMultiplicationByOne');
 const simplifyDoubleUnaryMinus = require('./simplifyDoubleUnaryMinus');
 
 const SIMPLIFICATION_FUNCTIONS = [
+  // convert mixed numbers to improper fractions
+  convertMixedNumberToImproperFraction,
```

Comment:
I think first is reasonable - there are some things in simplifyBasics that could go before this, but the basics functions include 1* ___ => ___ and that'd mess up some mixed numbers so let's just keep this at the top